### PR TITLE
2.2 announcement post

### PR DESCRIPTION
**Pull request by JonnyBurger**

### CODE REVIEW

File: packages/docs/blog/2021-07-09-remotion-2-2.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+author_image_url: https://avatars2.githubusercontent.com/u/1629785?s=460&u=12eb94da6070d00fc924761ce06e3a428d01b7e9&v=4
+---
+
+import { ThreeDPlayer } from "../components/3DPhonePlayer.tsx";
+
+Welcome to the release notes of Remotion 2.2! It's been a while since v2.1, but in the meanwhile we had a dozen smaller releases inbetween. This post summarizes the highlights of the past 2 months 🎉.
+
+## Environment variable support
+
+Us developers use environment variables extensively to manage configuration, secrets and other stuff you don't want to commit to a Github repo. If you write videos with code, environment variables are useful too!
```
Comment (MehmetAdemi) [inline packages/docs/blog/2021-07-09-remotion-2-2.md]: *We* developers ...

File: packages/docs/blog/2021-07-09-remotion-2-2.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+We added support for environment variables from the CLI, using a `.env` file, and allowing you to pass variables via our Node.JS APIs too. [Click here to learn more.](/docs/env-variables)
+
+## `<Freeze />` component
+
+This new core component will freeze all of it's children and make them think that the video is paused at a certain time.
+
+Use it for example to display a still frame from a video:
```
Comment (MehmetAdemi) [inline packages/docs/blog/2021-07-09-remotion-2-2.md]: *You can* use it for example ...

File: packages/docs/blog/2021-07-09-remotion-2-2.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+export const MyVideo = () => {
+  return (
+    <Freeze frame={30}>
+      <Video src={myVid} />
+    </Freeze>
+  )
+}
+```
+
+However, it works for any content. As you can see the API is dead simple. You can combine it with the [`<Sequence>`](/docs/sequence) API to make any content play, pause, and then continue again.
```
Comment (MehmetAdemi) [inline packages/docs/blog/2021-07-09-remotion-2-2.md]: ... any content play, pause and then continue again. (I'm not sure but maybe without a comma after "pause")

File: packages/docs/blog/2021-07-09-remotion-2-2.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+Learn more about the `playbackRate` props for [`<Audio />`](/docs/audio/#controlling-playback-speed) and [`<Video />`](/docs/video/#controlling-playback-speed).
+
+## `@remotion/three` package
+
+In case you missed it, we released a new helper package for [React Three Fiber](https://github.com/pmndrs/react-three-fiber)!
+
+<ThreeDPlayer />
+<br/>
+
+Three.JS is admittedly not easy, but with the work that [Poimandres](https://github.com/pmndrs) is doing in making it more approachable and more integrated with React, this is changing rapidly. Creating 3D videos in React and rendering them to real MP4 videos, we are making it possible!
```
Comment (MehmetAdemi) [inline packages/docs/blog/2021-07-09-remotion-2-2.md]: Three.JS is admittedly not easy, but with the work that [Poimanders] is doing *to make* it more approachable and more integrated with React, this is changing rapidly.

File: packages/docs/blog/2021-07-09-remotion-2-2.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+Learn more about [`@remotion/three`](/docs/three) and get started using our [template](https://github.com/remotion-dev/template-three).
+
+## ProRes support and MKV support
+
+Creating MP4 videos with React, that's just our marketing tagline. We support a lot of other codecs too, [like H265, WebM even transparent ones!](/docs/encoding)
+
+We have now added two more:
+
+- ProRes is a codec suitable for video editors using Final Cut Pro, Adobe Premiere or Davinci Resolve. Say you want to create an overlay animation using Remotion and use it in a traditional video cutting program, you can now use ProRes to export the video with an alpha channel and import it many other programs losslessly.
```
Comment (MehmetAdemi) [inline packages/docs/blog/2021-07-09-remotion-2-2.md]: [...], you can now use ProRes to export the video with an alpha channel and import it **to** many other programs losslessly.

File: packages/docs/blog/2021-07-09-remotion-2-2.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+- Fix Remotion Player assuming it's in a rendering environment
+- Fix getAudioMetadata() and getAudioDuration() not returning an error in the `onError` event (1ea7b151)
+- Added documentation on how to use TailwindCSS
+- Fixed documentation about how to use SASS/SCSS
+- If you are using a plain JS entry file, you can also use a remotion.config.js config file.
+- You can customize how many timeline tracks should be shown at most using `Config.Preview.setMaxTimelineTracks()`
+- Fixed a bug where environment variables would not be immediately accessible
+- Input props flag can now also be passed for preview
+- Better input validation of the most common functions
+- Improve rendering of very long videos, using less memory and avoiding a Maximum callstack exceeded error message.
```
Comment (MehmetAdemi) [inline packages/docs/blog/2021-07-09-remotion-2-2.md]: [...] avoiding a *maximum* callstack exceeded error message.

**Review by MehmetAdemi — COMMENTED** [review]
Hi Jonny, these are just a few recommendations. Overall it's easy to read and understand. It's a good article. Thumbs up!

**Review by MehmetAdemi — APPROVED** [review]

**Comment by Iamshankhadeep** [comment]
@JonnyBurger I just updated about th changes @MehmetAdemi suggested. Btw the blog post is awesome.

**Comment by JonnyBurger** [comment]
Thanks guys 🔥

**Pull request merged**